gui/schedule_gui.cc

```diff
@@ -1130,6 +1160,46 @@ void schedule_gui_t::init_line_selector()
 }
 
 
+void schedule_gui_t::init_next_line_selector()
+{
+	next_line_selector.clear_elements();
+	uint16 selection = 0;
+	vector_tpl<linehandle_t> lines;
+	linehandle_t temp_next_line; 
```

Comment:
特にtemporaryの意味はなさそうです
```suggestion
	linehandle_t next_line; 
```